### PR TITLE
Add support for ReadyMeters

### PR DESCRIPTION
**Pull request by Raphexion**
Add support for a Ready Meter

### CODE REVIEW

File: lib/wise_homex/models/ready_meter.ex [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  @moduledoc false
+
+  use WiseHomex.BaseModel
+
+  embedded_schema do
+    belongs_to :ready_installation, WiseHomex.ReadyInstallation
+    belongs_to :device, WiseHomex.Device
+
+    field :external_id, :integer
+    field :consumptiontype, :string
```
Comment (skovmand-wh) [inline lib/wise_homex/models/ready_meter.ex:11]: Shouldn't this be `consumption_type` for consistency in the API? Or is it already merged? :thinking: 
(The same for `serialnumber` and `metertype` below....)

**Review by skovmand-wh — APPROVED** [review]
Looks good, but I have a question about field names below.. Of course they have to match the API fields, but I think it makes sense that underscores are included whenever they should be.

File: lib/wise_homex/models/ready_meter.ex (hunk 1 above)
Comment (skovmand-wh) [reply lib/wise_homex/models/ready_meter.ex:11]: Or is it Kamstrup-specific field names?

Comment (Raphexion) [reply lib/wise_homex/models/ready_meter.ex:11]: I've taken the attributes directly from Kamstrup.

Comment (Raphexion) [reply lib/wise_homex/models/ready_meter.ex]: @skovmand Really good feedback. I've changed it both here and on server side.

**Pull request merged**